app/lib/services/voip/getInitialEvents.ts

```
@@ -0,0 +1,202 @@
+					);
+					await mediaSessionInstance.answerCall(data.callUUID);
+				} catch (error) {
+					console.error(`[VoIP][${platform}] Error handling call action event:`, error);
+				}
+			})
+		);
+	}
+
+	// Return cleanup function
+	return () => {
+		subscriptions.forEach(sub => sub.remove());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)